src/ruleset/v3/ruleset.ts

```diff
@@ -24,5 +25,18 @@ export const v3CoreRuleset = {
         function: operationMessagesUnambiguity,
       },
     },
+    'asyncapi3-required-operation-channel-unambiguity': {
+      description: 'Required operation (under root channels) "channel" must reference to a required channel (under root channels).',
```

Comment:
Moving the suggestion made by @jonaslagoni in the previous (closed) [PR](https://github.com/smoya/parser-js/pull/5#discussion_r1406717397).

```suggestion
      description: 'Operation channel's must reference to a required channel  (under root channels).'',
```

Reply:
But this only applies to those operations defined as required (under root). So I still think we should clarify this.
I agree my sentence is not so good though.

Reply:
Actually, I would even include those clarifications in the final error message the user will see, which is this one https://github.com/asyncapi/parser-js/pull/913/files#diff-e325fe55145146570971d2b1efd54035bbe2b005e557575279284a6f792c747bR35

Reply:
Two things:
1. Don't apply this suggestion as is because that won't be valid JS. See the conflict with single quotes.
2. Let's make it explicit everywhere. No need to use optional and required words. It could be something like `The "channel" field of an operation under the root "operations" object must always reference a channel under the root "channels" object.`. Same for https://github.com/asyncapi/parser-js/pull/913/files#diff-e325fe55145146570971d2b1efd54035bbe2b005e557575279284a6f792c747bR35.

Reply:
> The "channel" field of an operation under the root "operations" object must always reference a channel under the root "channels" object.

I like it